src/Lib/Modes/PathEditMode.vala

```diff
@@ -39,7 +55,8 @@ public class Akira.Lib.Modes.PathEditMode : AbstractInteractionMode {
     private int live_idx;
 
     // This flag tells if we are adding a path or editing an existing one.
-    private bool is_edit_path = true;
+    // Basically decides is we are in "edit" mode or "create" mode.
+    private bool is_edit_path = false;
```

Comment:
Probably should be an enum, since there will probably be more submodes for path editing

Reply:
create mode should probably just be called append or add